driver/src/main/java/org/neo4j/driver/v1/RecordAccess.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2002-2015 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.neo4j.driver.v1;
+
+import java.util.List;
+
+/**
+ * Access the fields of an underlying ordered map like data structure by key and index
+ */
+public interface RecordAccess extends ListAccess, MapAccess
```

Comment:
I don't like the `XXAccess` naming - that makes it sound like this is an interface for accessing `Records` (eg. something that has `Record get();` as a method.

Why not name it `XXView`?